tests/garage/envs/test_normalized_gym.py

```diff
@@ -18,8 +18,18 @@ def test_flatten(self):
             env.reset()
             for e in range(5):
                 env.render()
-                action = env.action_space.sample()
-                next_obs, reward, done, info = env.step(action)
+                a = env.action_space.sample()
+                a_copy = a if isinstance(a, (int, float)) else a.copy()
+                next_obs, _, done, _ = env.step(a)
+
+                # Check for side effects
+                if isinstance(a, (int, float)):
+                    assert a == a_copy,\
```

Comment:
PEP8 says to split lines with `()`, not `\`

Reply:
This doesn't test flattening at all and is therefore a different test function (e.g. `test_does_not_modify_action()`. If they share code, use `setUp()` to separate them.

Test code should be as granular as possible. When there's a failure, you want to see the exact few lines of code which make up the failing test, not a 100 line test function with 10 possible broken tests inside of it.